esp/src/src-react/components/Queries.tsx

```diff
@@ -25,18 +26,12 @@ const FilterFields: Fields = {
 };
 
 function formatQuery(filter: any, wuid?: string) {
```

Comment:
Doesn't seem to be using the wuid param any longer. Can it be removed?

Reply:
Agree - will fix.